num/src/smallint.rs

```diff
@@ -103,6 +105,48 @@ macro_rules! construct_smallint {
             }
         }
 
+        impl DivRem for $ty {
+            // divmod like operation, returns (quotient, remainder)
+            #[inline]
+            fn div_rem(self, other: Self) -> (Self, Self) {
+                //quotient and remainder will always be smaller than self so they're going to be in bounds
+                let quotient = self / other;
+                (quotient, self - (quotient*other))
+            }
+                /*let mut sub_copy = self;
```

Comment:
Can you pls explain why you have commented this implementation? Is it for the future use to have DivRem for different types? Can you than pls leave some comment right in the code explaining this intent so other contributors may work on that in the future?

Reply:
Oh, it's was a mistake, i copy the code from BigInt and forget to delete